src/main/java/Duke.java

```diff
@@ -7,12 +7,18 @@
 import java.time.format.DateTimeFormatter;
 import java.time.temporal.ChronoUnit;
 
+
```

Comment:
Do you think adding a JavaDoc description here would allow other users to better understand what exactly Duke does?